src/k8s-engine/wsl.ts

```diff
@@ -1093,8 +1093,21 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
         await this.progressTracker.action(
           this.lastCommandComment,
           100,
-          this.k3sHelper.updateKubeconfig(
-            async() => await this.captureCommand(await this.getWSLHelperPath(), 'k3s', 'kubeconfig')));
+          async() => {
+            // Wait for the file to exist first, for slow machines.
+            const command = 'if test -r /etc/rancher/k3s/k3s.yaml; then echo yes; else echo no; fi';
+
+            while (true) {
```

Comment:
`lima.ts` doesn't have a timeout on its loop waiting for `k3s.yaml` so I assume no timeout check is needed here either.

Reply:
Unfortunately it seems like people have slow enough machines that timeouts can generally be a bad idea unless they're unreasonably long :(